cypress/selectors/agenda.selectors.js

```diff
@@ -10,7 +10,7 @@ const selectors = {
   flatpickrDay: '.open  .flatpickr-day',
   button: 'button',
   overviewTitle: '[data-test-agendas-title]',
-  agendaListListItemDocuments: '[data-test-agenda-list-list-item-documents]',
+  agendaListListItemDocuments: '[data-test-agenda-overview-agenda-overview-item-documents]',
```

Comment:
Deze selector wordt nergens gebruikt in de templates (zowel de oude als de nieuwe)
Is het dan niet beter die gewoon weg te doen?